include/tinyara/lcd: Modify new LCD init code

seokhun-eom24 opened the pull request:
Modify new LCD init code for 2.8inch st7785.
Tested with lcd_test.

sunghan-chang reviewed (commented):
Could you leave what changed values mean?

seokhun-eom24 commented:
> Could you leave what changed values mean?

1. 0xB7 – Gate Control
Previous: VGH 14.06V, VGL -12.5V
Updated: VGH 14.06V, VGL -7.67V
By reducing the absolute value of VGL, pixels can turn off more easily.

2. 0xBB – VCOMS Setting
Previous: VCOM 0.75V
Updated: VCOM 0.95V
VCOM has been increased. LCD pixels toggle around this voltage (both positive and negative).

3. 0xC3 – VRH Set
Previous: VAP(GVDD): 4.3+( vcom+vcom offset+vdv) / VAN(GVCL): -4.3+( vcom+vcom offset-vdv)
Updated: VAP(GVDD): 4.5+( vcom+vcom offset+vdv) / VAN(GVCL): -4.5+( vcom+vcom offset-vdv)
Increasing the VRH reference voltage increases amplitude of gray scale voltage.

4. 0xE0, 0xE1 – Positive Voltage Gamma Control, Negative Voltage Gamma Control
Adjusting these gamma correction register value, these fine tunes the gamma curve. These values are used for calculating gamma level with complex formula.

Conclusion
If VGH, VGL, VCOM are unstable, it will cause image display failures, such as blurred screens, ghosting/sticking image, white screens, etc.
VRH and Positive Voltage Gamma Control / Negative Voltage Gamma Control is related to gray scale.
These value changes are intended to adjust the display and grayscale control voltage settings to perfectly fit the LCD.

Terminology
VGH: high potential of the gate level, which is the voltage that opens the gate level.
VGL: low potential of the gate stage, which is the voltage that turns off the gate stage.
VCOM: reference voltage for deflection of liquid crystal molecules
VAP: A power output of grayscale voltage generator.
VAN: A power output (Negative) of grayscale voltage generator.
